src/plugins/data/public/query/state_sync/sync_state_with_url.ts

```diff
@@ -85,7 +85,10 @@ export const syncQueryStateWithUrl = (
     stateContainer: {
       ...globalQueryStateContainer,
       set: state => {
-        globalQueryStateContainer.set(state || defaultState);
+        if (typeof state !== 'object' || state === null) {
```

Comment:
What do you want to check here? 
`typeof null -> 'object'`
Do you want to check for `undefined`?

Reply:
i want to be sure that only an object comes through, and since `typeof null -> object` as you said, so I've checked it that way

Reply:
Got it, thanks. But from my point of view it's enough to check for null like we do e.g. `src/legacy/core_plugins/kibana/public/visualize/np_ready/editor/lib/visualize_app_state.ts`

```
  const { start: startStateSync, stop: stopStateSync } = syncState({
    storageKey: STATE_STORAGE_KEY,
    stateContainer: {
      ...stateContainer,
      set: state => {
        if (state) {
          // syncState utils requires to handle incoming "null" value
          stateContainer.set(state);
        }
      },
    },
    stateStorage: kbnUrlStateStorage,
  });
```

@Dosant could you please confirm that?

Reply:
`null` is enough. null either means that there is no `_a` in the url at all or if can't parse whatever is in `_a`.

Reply:
thx, changed that